Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM public.ecr.aws/m5j1b6u0/zinc:latest AS builder
```

Comment:
The image currently at `jnsgruk:latest` is using `FROM `public.ecr.aws/h9e2j3o7/zinc:0.1.9 AS builder` as a base.